app/models/user.rb

```diff
@@ -64,6 +64,10 @@ def confirmed?
     email_addresses.where.not(confirmed_at: nil).any?
   end
 
+  def accepted_rules_of_use?
+    self.accepted_terms_at.present?
```

Comment:
is the plan to do something like `self.accepted_terms_at > Date.new(2021, 1, 1)` if we update them in the future?

Reply:
cool, makes sense!